src/main/org/firebirdsql/jdbc/AbstractResultSet.java

```diff
@@ -128,7 +131,7 @@ public AbstractResultSet(FBConnection connection,
                         .toFlatSQLException(SQLWarning.class));
                 rsType = ResultSet.TYPE_SCROLL_INSENSITIVE;
             }
-
+            closeableFields = null;
```

Comment:
Unnecessary initalization, fields are always initialized to null if not explicitly assigned